db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :host_name
+      t.string :status
+      t.string :scheduling_status
+      t.string :zone_name
+      t.string :disabled_reason
+      t.bigint :ems_id
+      t.references :host, :foreign_key => true
+      t.references :system_service, :foreign_key => true
+      t.references :availability_zone, :foreign_key => true
```

Comment:
Please remove the foreign_key bits...we don't use low-level foreign keys in ManageIQ.  Also, be sure the `:type => :bigint` is set on all of these for region support.